pages/linux/repo-add.md

```diff
@@ -0,0 +1,15 @@
+# repo-add
+
+> Package database maintenance utility which enables installation via pacman.
+
+- Add all package binaries in the current directory and remove the old database file:
```

Comment:
Add all package binaries in the current directory to where?

Reply:
To a database file in the same directory as shown in the following image. To put it up 1 directory, you'd run ../yourdatabasename.db.tar.gz
![image](https://user-images.githubusercontent.com/60808802/135693525-324c1643-91c4-46cd-8ad1-ed866320084e.png)